<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SDialogEditAddress</class>
 <widget class="QDialog" name="SDialogEditAddress">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>修改地址</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="horizontalSpacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>等距更纱黑体 SC</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>描述:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="txtDesc">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>等距更纱黑体 SC</family>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>等距更纱黑体 SC</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>类型:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="cbType">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>等距更纱黑体 SC</family>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>等距更纱黑体 SC</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>地址:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="txtAddress">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>等距更纱黑体 SC</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="txtValue">
       <property name="font">
        <font>
         <family>等距更纱黑体 SC</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>=??</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="cbBase16">
       <property name="font">
        <font>
         <family>等距更纱黑体 SC</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>十六进制</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbSign">
       <property name="font">
        <font>
         <family>等距更纱黑体 SC</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>有符号</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="font">
        <font>
         <family>等距更纱黑体 SC</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QGroupBox" name="GroupPointer">
     <property name="font">
      <font>
       <family>等距更纱黑体 SC</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>指针</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="txtPAddress">
          <property name="minimumSize">
           <size>
            <width>210</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>210</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>等距更纱黑体 SC</family>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="txtPtrTo">
          <property name="font">
           <font>
            <family>等距更纱黑体 SC</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>-&gt;????????</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ButtonAppendOffset">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>等距更纱黑体 SC</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>添加偏移</string>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="font">
         <font>
          <family>等距更纱黑体 SC</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <spacer name="verticalSpacer_2">
     <property name="font">
      <font>
       <family>等距更纱黑体 SC</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>等距更纱黑体 SC</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="ButtonAccept">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>等距更纱黑体 SC</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonCancel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>等距更纱黑体 SC</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
